requirements.txt

```diff
@@ -1,5 +1,6 @@
 boto3>=1.9.51
 botocore>=1.12.51
+graphviz>=0.10.1
```

Comment:
can we make this an optional requirement? 

my recommended approach would be to import inside of `plot` some like this 

```
try:
    import graphviz
except ImportError:
    raise ImportError('Please install graphviz to install use plotting functionality')
```

Reply:
Fair enough. It's now solved with a779771